<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prod_edit</class>
 <widget class="QWidget" name="prod_edit">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>138</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>273</width>
    <height>138</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>273</width>
    <height>138</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Ціна</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Змінити</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
     <property name="maximum">
      <double>999999.989999999990687</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Ціна, опт</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
     <property name="maximum">
      <double>999999.989999999990687</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Ціна, бартер</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="maximum">
      <double>999999.989999999990687</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Ціна, роздр</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
